### PR TITLE
Quotes

### PR DESCRIPTION
**Pull request by crazybmanp**

### CODE REVIEW

**Review by razage — CHANGES_REQUESTED** [review]
redo this with subcommands

**Comment by crazybmanp** [comment]
Now ready for new review.

File: OWext/__init__.py [hunk 1]
```diff
@@ -7,22 +7,24 @@
 
 
 class OWext():
```
Comment (razage) [inline OWext/__init__.py:9]: redundant parenthesis

File: Quotes/__init__.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    async def delete(self, ctx, quotenum):
+        """Deletes the quote at the given quote index.
+        :quotenum: The number of the quote to be deleted
+        """
+        sdb = self.server_db[ctx.message.server.id]
+        sdb.remove(eid=quotenum)
+        await self.bot.say("Removed quote #{}".format(quotenum))
+
+    @quote.command(pass_context=True)
+    async def list(self, ctx, category=None):
```
Comment (razage) [inline Quotes/__init__.py:85]: it's probably fine, but list is a reserved word.

**Review by razage — COMMENTED** [review]
None of the changes I requested affect functionality, but you might want to look at them.

**Review by razage — APPROVED** [review]

**Pull request merged**